jsonpb/jsonpb_test.go

```diff
@@ -684,3 +684,33 @@ func (m *dynamicMessage) UnmarshalJSONPB(jum *Unmarshaler, json []byte) error {
 	m.rawJson = string(json)
 	return nil
 }
+
+// customFieldMessage implements protobuf.Message but is not a normal generated message type.
+type customFieldMessage struct {
```

Comment:
This is similar to the dynamicMessage but w/o the MarshalJSONPB code.
The message has no other fields than the custom field that should be ignored.
The accompanying test ensures that the  JSON output is `{}` of this type